packages/config/src/options.ts

```diff
@@ -204,6 +204,11 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     validation: validate.isBoolean,
     isExperimental: true,
     requireRestartOnChange: 'server',
+  }, {
+    name: 'experimentalRunAllSpecs',
+    defaultValue: false,
+    validation: validate.isBoolean,
+    isExperimental: true,
```

Comment:
You'll need to update [@packages/server/lib/experiments](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/experiments.ts#L53) to include this option & description for the App setting page

Reply:
The App settings page was working without this addition, but look like it is used for run-mode terminal printing
https://github.com/cypress-io/cypress/blob/55f6972f1d4a43abccc32b501b2df0b2987f4c17/packages/server/lib/util/print-run.ts#L186